realm/realm-library/src/objectServer/java/io/realm/UserStore.java

```diff
@@ -68,4 +68,12 @@
      * @return Collection of all users. If no users exist, an empty collection is returned.
      */
     Collection<SyncUser> allUsers();
+
+    /**
+     * Returns the state of the specified user: {@code true} if active (not logged out), {@code false} otherwise.
```

Comment:
Perhaps also clarify that this will return true if the user has expired but not actively logged out.